### ISSUE TITLE
PanelMenu: Different item is focussed when item is pressed

### ISSUE THREAD

**Issue by vdbongard** [primefaces/primeng#18010]
### Describe the bug

If a PanelMenu item is actively being selected, another item in the PanelMenu group is focused as well.

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/panelmenu#template

### Environment

Chrome 134, Windows 11

### Angular version

19.2.0

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Go to the "Template" section of the PanelMenu docs (https://primeng.org/panelmenu#template)
2. Open the "Mail" group
3. Actively select the second entry of "Inbox" (hold the mouse down while clicking the item)
4. The first item "Compose" also gets highlighted, while pressing the second item "Inbox" 

![Image](https://github.com/user-attachments/assets/15355290-96ca-4d87-8fd0-6b8e45ebf5b7)

### Expected behavior

The first item should not get highlighted while pressing the second item.

**Comment by vdbongard** [comment]
The bug still exists. The error has changed slightly. When actively selecting the second entry of "Inbox" (step 3), the last item "Trash" will get highlighted. (instead of the first item "Compose" in step 4)

<img width="612" height="463" alt="Image" src="https://github.com/user-attachments/assets/e5585daa-9b5c-4734-80cc-ee2d52e73d4e" />

**Comment by bae-sh** [comment]
Same issue as #13940, but the error still continues.
When I run PanelMenu in dev mode, it works well, but in prod mode, this issue happens.
@cagataycivici Can I work on this issue?

**Comment by bae-sh** [comment]
@cagataycivici I've fixed the bug and opened PR #19296. Could you please review it?

**Comment by cetincakiroglu** [comment]
Hi,

This issue already fixed in here and the fix will available with upcoming release. https://github.com/primefaces/primeng/commit/2f431599c6a27d7cc382449efe8bbc2caa3d2b6f#diff-96a09a58c7c12744d56b6a66388fd0da4643399cf3739accc097bf025f616495R159

**Comment by vdbongard** [comment]
I have just tested this. The bug still exists like it is described in the issue.

Environment
Chrome 134, Windows 11

PrimeNG version
v21

I created a new issue as I can't reopen this.

https://github.com/primefaces/primeng/issues/19526
